lite/pom.xml

```diff
@@ -550,6 +550,11 @@
                         <include>fix_throws_javadoc.sh</include>
                         <!-- all source files in a directory named util, in case the samples start picking those up -->
                         <include>src/**/util/**/*</include>
+                        <!-- Maven Wrapper files -->
+                        <include>.mvn/wrapper/maven-wrapper.jar</include>
```

Comment:
The ``**`` means not only all files, but all _directories_, recursively. So the fileglob I mentioned matches everything under ``.mvn/wrapper``, recursive into all directories. The syntax in Java is most commonly expressed in Ant: http://ant.apache.org/manual/dirtasks.html

Just plain ``*`` matches only files, ignoring directories.